packages/tokamak/sdk/tasks/deposit-erc20.ts

```diff
@@ -17,6 +16,7 @@ import Artifact__OptimismPortal from '@tokamak-network/thanos-contracts/forge-ar
 import Artifact__L1CrossDomainMessenger from '@tokamak-network/thanos-contracts/forge-artifacts/L1CrossDomainMessenger.sol/L1CrossDomainMessenger.json'
 import Artifact__L1StandardBridge from '@tokamak-network/thanos-contracts/forge-artifacts/L1StandardBridge.sol/L1StandardBridge.json'
 import Artifact__L2OutputOracle from '@tokamak-network/thanos-contracts/forge-artifacts/L2OutputOracle.sol/L2OutputOracle.json'
+import Artifact__WETH9 from '@tokamak-network/thanos-contracts/forge-artifacts/WNativeToken.sol/WNativeToken.json'
```

Comment:
How about changing from Artifact__WETH9 to Artifact__WTON?

Reply:
WNativeToken!